pkg/klusterlet/clusterinfo/loggingInfo_sync.go

```diff
@@ -328,3 +341,27 @@ func subHostName(agentNamespace string) string {
 	// the length of hash is 32, `workmgr-<hash of sub name>` is 40
 	return fmt.Sprintf("workmgr-%s", hashOfString(subName))
 }
+
+func (s *loggingInfoSyncer) cleanup(ctx context.Context) error {
```

Comment:
not quite sure why we want to delete service and route here.

Reply:
deleted cleanup